add product feature user feedback content (#3801)
getsentry/sentry-docs: src/docs/product/user-feedback/index.mdx

---
title: User Feedback
sidebar_order: 70
description: "Learn how you can view user feedback submissions, paired with the original event, giving you additional insight into issues."
---

> While event data allows you to collect information about errors in your application, getting any extra helpful feedback can be crucial in determining the root cause of an issue. Sentry provides the ability to collect additional information with User Feedback. The **User Feedback** page in [sentry.io](https://sentry.io) provides you with a list of feedback submissions from users who experienced an error while using your application. Each submission includes the name and email address entered by the user, as well as their description of what happened to produce the error.

Reviewer: ```suggestion
While event data allows you to collect information about errors in your application, getting any extra helpful feedback can be crucial in determining the root cause of an issue. Sentry provides the ability to collect additional information with User Feedback. The **User Feedback** page in [sentry.io](https://sentry.io) displays feedback submissions from users who experienced an error while using your application. Each submission includes the name and email address entered by the user, as well as their description of what happened to produce the error.
```

![User Feedback page with submissions.](user-feedback.png)
<!--placeholder image to be updated-->

In each submission, you can click the “View Event” link, which takes you to the **Issue Details** page for that event (or instance of the issue). From there, you can dig into what caused the issue and start working to resolve it.

![The event that generated the user feedback on the Issue Details page.](user-feedback-event.png)
<!--placeholder image to be updated-->

## Set Up

> Sentry provides two ways to collect user feedback. If you’re using Sentry's JavaScript, you can configure [an embeddable widget](/platforms/javascript/enriching-events/user-feedback/) that displays when a user experiences an error. If you’d prefer to use an alternative widget, or you don’t have a JavaScript frontend, you can use the [User Feedback API](https://docs.sentry.io/api/projects/submit-user-feedback/) instead.

Reviewer: We document all methods on the User Feedback page (the content varies according to what the SDK supports - here's what we display for [Android](https://docs.sentry.io/platforms/android/enriching-events/user-feedback/) v. what we display for [JavaScript](https://docs.sentry.io/platforms/javascript/enriching-events/user-feedback/) v. what we display for .[NET](https://docs.sentry.io/platforms/dotnet/guides/aspnet/enriching-events/user-feedback/#integration), for example), so suggest we edit to:
```suggestion
Sentry provides two ways to collect user feedback, an API or an embeddable widget, which you can [configure]((/platform-redirect/?next=/enriching-events/user-feedback/) to collect feedback when a user experiences an error. 
```

---
title: User Feedback
sidebar_order: 70

> description: "Learn how you can view user feedback submissions, paired with the original event, giving you additional insight into issues."

Author: ```suggestion
description: "Learn how you can view user feedback submissions which, paired with the original event, give you additional insight into issues."
```

---

While event data allows you to collect information about errors in your application, getting any extra helpful feedback can be crucial in determining the root cause of an issue. Sentry provides the ability to collect additional information with User Feedback. 

The **User Feedback** page in [sentry.io](https://sentry.io) displays feedback submissions from users who experienced an error while using your application. Each submission includes the name and email address entered by the user, as well as their description of what happened to produce the error.

![User Feedback page with submissions.](user-feedback.png)
<!--placeholder image to be updated-->

In each submission, you can click the “View Event” link, which takes you to the **Issue Details** page for that event (or instance of the issue). From there, you can dig into what caused the issue and start working to resolve it.

![The event that generated the user feedback on the Issue Details page.](user-feedback-event.png)
<!--placeholder image to be updated-->

## Set Up

> Sentry provides two ways to collect user feedback, an API or an embeddable widget, which you can [configure]((/platform-redirect/?next=/enriching-events/user-feedback/) to collect feedback when a user experiences an error. 

Author: ```suggestion
Sentry provides two ways to collect user feedback, an API or an embeddable widget, which you can [configure](/platform-redirect/?next=/enriching-events/user-feedback/) to collect feedback when a user experiences an error. 
```
